src/moonlink_connectors/src/pg_replicate/conversions.rs

```diff
@@ -52,4 +53,5 @@ pub enum ArrayCell {
     Uuid(Vec<Option<Uuid>>),
     Json(Vec<Option<serde_json::Value>>),
     Bytes(Vec<Option<Vec<u8>>>),
+    Composite(Vec<Option<Vec<Cell>>>),
```

Comment:
nit: suggest to add some comments, the type is very complicated and not easy to understand